packages/bpk-component-banner-alert/src/BpkBannerAlertInner.js

```diff
@@ -0,0 +1,257 @@
+/*
```

Comment:
Is this file mostly just a rename?

Reply:
Yeah and the addition of the `configuration` prop